### DISCUSSION TITLE
Strech and Skew

### DISCUSSION THREAD

**Discussion by blpraveen** [fabricjs/fabric.js#9729] [Ideas]
How can I stretch and skew the seats.

I am referring to code here [codepen link](https://codepen.io/trirandom/pen/poZdagp)

This code only transforms the box.

In this [seatmap.pro](seatmap.pro) website it allows the seats inside a bounding box to stretch and skew.

I need help to convert codepen link code above to stretch and skew

**Comment by asturur** [comment]
@blpraveen this is not the way to start nor a discussion nor an issue.

**Reply by blpraveen** [reply]
Is my question now understandable?
